test/servico/cst03Test.java

```diff
@@ -76,8 +76,8 @@ public void tearDown() {
             exceptionOk = true;
         }
         
-        assertTrue(loginService.VerificarUsuarioESenha("Administrador", "123456"));
-        ausencias = notificaçãoService.listarAusencias();        
+        assertTrue(loginService.verificarUsuarioESenha("Administrador", "123456"));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 83).](https://www.codacy.com/app/calebepb/substituicao/file/2738501513/issues/source?bid=3308597&fileBranchId=3353492#l79)